recipes-devtools/python/python3-ryu/0003-ryu-lib-ofctl_v1_3.py-add-ofdpa-match-fields.patch

```diff
@@ -28,7 +28,7 @@ index 91ae9a0f..fd908cd3 100644
                 'pbb_isid': ofctl_utils.to_match_masked_int,
                 'tunnel_id': ofctl_utils.to_match_masked_int,
 -               'ipv6_exthdr': ofctl_utils.to_match_masked_int}
-+               'ipv6_exthdr': ofctl_utils.to_match_masked_int
++               'ipv6_exthdr': ofctl_utils.to_match_masked_int,
```

Comment:
This is the reason I prefer to also end the last line with a comma in arrays/lists, but alas (IIRC) python doesn't allow it.